### PR TITLE
Telco VKACI changes

### PR DESCRIPTION
**Pull request by burhan20**

### CODE REVIEW

File: app/graph.py [hunk 1]
```diff
@@ -785,15 +791,36 @@ def __init__(self, env: VkaciEnvVariables, topology: VkaciBuilTopology) -> None:
...
     WITH $json as data
     UNWIND data.items AS n
     UNWIND n.node_leaf_sec_iface_conn AS conn
+    WITH n, conn, CASE WHEN n.vm_hosts IS NOT NULL AND size(n.vm_hosts) = 0 THEN true ELSE false END AS vm_hosts_empty WHERE vm_hosts_empty
     MATCH (node:Node) WHERE node.name = n.node_name
     MATCH (switch:Switch) WHERE switch.name = conn.switch_name
     MERGE (node)-[:CONNECTED_TO_SEC {interface: conn.node_iface + " : " + conn.switch_interface}]->(switch)
     """
 
+    query33 = """
```
Comment (abhis2112) [inline app/graph.py:814]: change these names query33/query22

File: app/graph.py [hunk 2]
```diff
@@ -859,39 +909,56 @@ def update_database(self):
...
-                        "interface": " | ".join(list(interfaces))
-                    })
-                    switch_items[switchName]["vm_hosts"].append(neighbour)
+                    logger.info("BURHAN: %s, %s", node, neighbour)
+                    if baremetal:
+                        switch_items[switchName]["nodes"].append({
+                            "name": node,
+                            "interface": " | ".join(list(interfaces))
+                        })
+                        logger.info("BURHAN1: %s, %s %s", node, neighbour, switch_items)
```
Comment (abhis2112) [inline app/graph.py:944]: remove irrelevant logs

File: app/template/assets/cui-2.0.0/js/vkaci.js [hunk 3]
```diff
@@ -258,28 +258,46 @@ function draw_only_bgp_peers() {
 
 function draw_only_primary_links() {
     selectedView = View.OnlyPrimarylinks
+    // let q = `OPTIONAL MATCH (p:Pod)-[r:RUNNING_ON]->(n:Node) WHERE p.ns =~ '${selectedNamespace}'
```
Comment (abhis2112) [inline app/template/assets/cui-2.0.0/js/vkaci.js:261]: remove commented codes

File: app/graph.py (hunk 1 above)
Comment (burhan20) [reply app/graph.py:814]: yes, I will change.

File: app/graph.py (hunk 2 above)
Comment (burhan20) [reply app/graph.py:944]: i will remove

File: app/template/assets/cui-2.0.0/js/vkaci.js (hunk 3 above)
Comment (burhan20) [reply app/template/assets/cui-2.0.0/js/vkaci.js:261]: i will remove

**Pull request closed without merging**